Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -907,10 +905,18 @@ extension CarPlayManager: CPMapTemplateDelegate {
     public func mapTemplate(_ mapTemplate: CPMapTemplate,
                             didUpdatePanGestureWithTranslation translation: CGPoint,
                             velocity: CGPoint) {
-        updatePan(by: translation, mapTemplate: mapTemplate, animated: false)
+        // Map view panning is allowed in all states except routes preview.
+        if currentActivity == .previewing {
```

Comment:
Do not allow panning while previewing routes.

Reply:
It's just makes thinks simpler in first place (It not hard to enable it in routes preview). If we also add it we'll have to provide additional buttons to recenter map etc. For first iteration I think it'd be better to simplify things as much as possible.